modules/arm_plugin/README.md

```diff
@@ -0,0 +1,212 @@
+In order to build the plugin, you must prebuild OpenVINO package from source using [this guideline](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode#building-for-different-oses).
+
+Afterwards plugin build procedure is as following:
+
+1.  Install necessary build dependencies:
+```
+sudo apt-get update  
+sudo apt-get install -y git cmake  scons build-essential
+```
+2. Clone `openvino_contrib` repository:
+```
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino_contrib.git 
+```
+3.  Go to plugin directory:
+ ```
+cd openvino_contrib/modules/arm_plugin
+```
+4.  Prepare a build folder:
+```
+mkdir build && cd build
+```
+5.  Build plugin:
+```
+cmake -DInferenceEngineDeveloperPackage_DIR=<path to OpenVINO package build folder> -DCMAKE_BUILD_TYPE=Release .. && make
+```
+
+## Sample
+You could verify the plugin by running [OpenVINO™ samples]. You can find C++ samples in `build` directory (if you build the plugin using approach #1) or `openvino/bin/armv7l/Release` directory (if you build the plugin using approach #2 or #3). The following procedure assumes the approach #1 is used.  
+OpenVINO samples require OpenCV libraries. If you build the plugin using approach #1 all needed OpenCV libraries are already placed in `build\lib` directory. If you build the plugin using approach #2 or #3 you need to install OpenCV or [build it from source].  
+Let's try to run [Object Detection for SSD sample].  
+### Model preparation
+To speed up the process you may prepare the model on non-ARM platform.
+
+1. Install [Model Optimizer]:
+```
+git clone https://github.com/openvinotoolkit/openvino.git
+cd openvino/model-optimizer
+pip3 install requirements.txt
+cd ../..
+```
+2. Install [model downloader] and [model converter] from Open Model Zoo:
+```
+git clone https://github.com/openvinotoolkit/open_model_zoo.git
+cd open_model_zoo/tools/downloader
+python3 -mpip install --user -r ./requirements.in
+```
+3. Download and convert model `vehicle-license-plate-detection-barrier-0123` using model downloader and model converter:
+```
+python3 ./downloader.py --name vehicle-license-plate-detection-barrier-0123 --precisions FP32
+python3 ./converter.py --mo ../../../openvino/model-optimizer/mo.py --name vehicle-license-plate-detection-barrier-0123 --precisions FP32
+```
+### Model inference on ARM
+1. Copy `build` directory with OpenVINO™ and ARM plugin artefacts to ARM platform.
+2. Go to `build` directory:
+```
+cd build
+```
+3. Download a vehicle image, for instance, [this image]:
+```
+wget https://raw.githubusercontent.com/openvinotoolkit/openvino/master/scripts/demo/car_1.bmp
+```
+4. Copy model [Intermediate Representation] (`vehicle-license-plate-detection-barrier-0123.bin` and `vehicle-license-plate-detection-barrier-0123.xml` files generated by model converter) to ARM platform.
+
+5. Run object detection sample on ARM platform:
+```
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:lib
```

Comment:
It is not recommended to use relative paths in `LD_LIBRARY_PATH` from security perspective:

```
$(pwd)/lib
```

Reply:
fixed the path